packages/www/package.json

```diff
@@ -11,8 +11,6 @@
     "postbuild": "next-sitemap",
     "type-check": "tsc --pretty --noEmit",
     "docs-positions": "node ./generate-docs-positions.js",
-    "docker:build": "docker build $(printf ' -t livepeerci/www:%s' ${DOCKER_TAGS:-master}) -f Dockerfile ../..",
```

Comment:
You can get rid of push maybe, but I've used this command to build the image locally in the past